src/Tribe/Views/V2/View.php

```diff
@@ -1426,6 +1430,7 @@ protected function setup_template_vars() {
 			'breakpoint_pointer'     => $this->get_breakpoint_pointer(),
 			'is_initial_load'        => $this->context->doing_php_initial_state(),
 			'public_views'           => $this->get_public_views( $url_event_date ),
+			'show_recent_past'       => $this->should_show_recent_past_events_view(),
```

Comment:
:warning: Array double arrow not aligned correctly; expected 5 space(s) between "'show_recent_past'" and double arrow, but found 7.

<sub>hash: cdf7e728c2cc672c2307533e86b317e9</sub>